helix-view/src/env.rs

```diff
@@ -0,0 +1,13 @@
+use std::fmt::Display;
+
+use crate::Document;
+
+pub fn inject_into_with<T>(strs: T, doc: &Document) -> Vec<String>
```

Comment:
The name is a bit awkward. I think something like `inject_envorinment` may fit better